drivers/tps6274x/tps6274x.c

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2017 RWTH Aachen, Steffen Robertz, Josua Arndt
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+#include "tps6274x.h"
+#define ENABLE_DEBUG        (0)
```

Comment:
Currently you're not using `DEBUG()` **but** I'll propose to use it some lines below. If you won't implement it, we should get rid of this define here.